src/pages/Login/Login.js

```diff
@@ -1,12 +1,114 @@
-import React from "react";
+import React, { useState } from 'react';
 import './Login.scss';
+import Logo from './logo.png';
+import { useNavigate } from 'react-router-dom';
 
 const Login = () => {
+  const navigate = useNavigate();
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+
+  const setChangeEmail = (event) => {
+    setEmail(event.target.value);
+  };
+
+  const setChangePassword = (event) => {
+    setPassword(event.target.value);
+  };
+
+  const goToSignUp = () => {
+    navigate('./sign-up');
+  };
```

Comment:
라우팅 경로는 절대경로로 작성하시면 됩니다.
```suggestion
  const goToSignUp = () => {
    navigate('/sign-up');
  };
```